docs/google/devices.md

```diff
@@ -0,0 +1,194 @@
+# Google Devices
+
+This section describes the devices in Cirq for Google hardware devices and their usage.
+Since quantum hardware is an active area of research, hardware specifications and best
+practices are constantly evolving in an attempt to continuously improve performance.
+While this information should be a solid base for beginning your quantum application,
+please work with your Google sponsor to obtain the latest information on devices that
+you plan to use. 
+
+## General limitations
+
+Qubits on Google devices are laid out in a grid structure.  Connectivity is limited to
+adjacent qubits, either horizontally or vertically.  
+
+Measurement takes much longer than other gates.  Currently, the only supported
+configuration is to have terminal measurement in the final moment of a circuit.
+
+Most devices have a limited number of gates that can applied to the device.  Gates not
+in that set must be decomposed into an equivalent circuit using gates within the set.
+See below for those restrictions. 
+
+## Best Practices
+
+This section lists some best practices for creating a circuit that performs well on hardware
+devices.  This is an area of active research, so users are encouraged to try multiple
+approaches to improve results.
+
+### Use built-in optimizers as a first pass
+
+Using built-in optimizers will allow you to compile to the correct .  As they are
+automated solutions, they will not always perform as well as a hand-crafted solution, but
+it provides a good starting point for creating a circuit that is likely to run successfully
+on hardware.
+
+```python
+import cirq
+import cirq.google as cg
+
+
+# Create your circuit here
+my_circuit = cirq.Circuit()
+
+# Convert the circuit onto a Google device.
+# Specifying a device will verify that the circuit satisifies constraints of the device
+# Specifying an optimizer type (e.g. 'sqrt_iswap' or 'sycamore') will convert to the appropriate
+# gate set and also do basic optimization, such as combining successive one-qubit gates. 
+sycamore_circuit = cg.optimized_for_sycamore(my_circuit, new_device=cg.Sycamore, optimizer_type='sqrt_iswap')
+```
+
+### Good moment structure
+
+In an attempt to fulfill the user's desire, quantum devices will respect the moment
+structure of circuits to the greatest extent possible.
+
+To this end, it is important that the moment structure of the circuit is kept as
+short and concise as possible.  The length of a moment will generally be the length
+of the longest gate in the moment, so keeping gates with similar durations together
+will generally benefit the performance.
```

Comment:
"help performance"

it benefits the user. performance don't care

Reply:
Changed to "...keeping gates with similar durations together
will shorten the duration of the circuit and likely reduce the noise incurred"